website/tables/aws/aws_costexplorer_cost_30d.md

```diff
@@ -1,6 +1,6 @@
-# Table: aws_costexplorer_cost_current_month
+# Table: aws_costexplorer_cost_30d
 
-This table shows data for AWS Cost Explorer Cost Current Month.
+This table shows data for AWS Cost Explorer Cost 30d.
```

Comment:
nit: To improve this you could change the `titleTransformer` in `resources/plugin/plugin.go` and add an override (`awsExceptions` is just for the service name) so it says "AWS Cost Explorer costs for the last 30 days" etc...

Reply:
No need for exceptions in this case as we can just hard code the title... Once we get more similar exceptions (beyond just the service name)...